### PR TITLE
 APIS-7251: Align with other API platform repos

### PR DESCRIPTION
**Pull request by kskourtis1**

### CODE REVIEW

**Review by mattclark-zerogravit — DISMISSED** [review]
👍

**Review by mattclark-zerogravit — APPROVED** [review]
👍

File: build.sbt [hunk 1]
```diff
@@ -1,21 +1,51 @@
...
   .enablePlugins(PlayScala)
   .dependsOn(microservice % "test->test")
   .settings(DefaultBuildSettings.itSettings())
-  .settings(libraryDependencies ++= AppDependencies.it)
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    headerSettings(Test) ++ automateHeaderSettings(Test),
+    inConfig(Test)(BloopDefaults.configSettings),
+    addTestReportOption(Test, "int-test-reports")
```
Comment (johnsgp) [inline build.sbt:38]: Please remove this line as it causes problems with test reports in Jenkins

File: build.sbt [hunk 2]
```diff
@@ -1,21 +1,51 @@
...
-lazy val it = project
+lazy val it = (project in file("it"))
   .enablePlugins(PlayScala)
   .dependsOn(microservice % "test->test")
   .settings(DefaultBuildSettings.itSettings())
-  .settings(libraryDependencies ++= AppDependencies.it)
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    headerSettings(Test) ++ automateHeaderSettings(Test),
```
Comment (johnsgp) [inline build.sbt:36]: This is no longer needed

**Review by petekirby-ee — DISMISSED** [review]

File: build.sbt [hunk 3]
```diff
@@ -1,21 +1,51 @@
...
+    inConfig(Test)(BloopDefaults.configSettings),
```
Comment (johnsgp) [inline build.sbt:37]: Bloop defaults are already Global at the top of this file, so this line is not needed
```

File: build.sbt (hunk 1 above)
Comment (mattclark-zerogravit) [reply build.sbt:38]: ok

**Review by petekirby-ee — APPROVED** [review]

**Pull request merged**